roles/falcon/defaults/main.yml

```diff
@@ -1,83 +1,73 @@
+# ensures instances spun up from the master receive their own, unique,
+# Falcon Agent ID.
+#
+# If installing this role on a base AMI or master image, set this flag
+# to 'true'.
+#
+falcon_remove_agent_id: false
 
-    #
-    # Where should the sensor file be downloaded to? By default
-    # this will be the homedir of the user the ansible playbook
-    # is running as.
-    #
-    # Note: On Linux, do not forget trailing "/", e.g. "/tmp/"!
-    #
-    install_temp_dir: ""
+# CrowdStrike API URL for downloading the Falcon sensor. Possible values:
+#       US-GOV-1:   api.laggar.gcw.crowdstrike.com
+#       EU-1:       api.eu-1.crowdstrike.com
+#       US-2:       api.us-2.crowdstrike.com
+#       Catch all:  api.crowdstrike.com
+#
+falcon_api_url: "api.crowdstrike.com"
 
-    #
-    # Falcon requires that a master image remove the Falcon Agent ID (AID). This
-    # ensures instances spun up from the master receive their own, unique,
-    # Falcon Agent ID.
-    #
-    # If installing this role on a base AMI or master image, set this flag
-    # to 'true'.
-    remove_agent_id: false
+# 'Client ID' and 'Client Secret' for authentication against the CrowdStrike
+# API. If unknown, download from the CrowdStrike support portal:
+#
+#   https://falcon.crowdstrike.com/support/api-clients-and-keys
+#
+falcon_api_client_id: ""
+falcon_api_client_secret: ""
 
-    #
-    # Your Falcon Customer ID (CID)
-    #
-    cid: yourCID123
+# Where should the sensor file be downloaded to? By default
+# this will be the homedir of the user the ansible playbook
+# is running as.
+#
+# Note: On Linux, do not forget trailing "/", e.g. "/tmp/"!
+#
+falcon_install_tmp_dir: ""
 
-    #
-    # PARAMETERS FOR API-BASED INSTALLATION
-    #
-    api:
-      #
-      # Which CrowdStrike API should we download from? Possible values:
-      #       US-GOV-1:   api.laggar.gcw.crowdstrike.com
-      #       EU-1:       api.eu-1.crowdstrike.com
-      #       US-2:       api.us-2.crowdstrike.com
-      #       Catch all:  api.crowdstrike.com
-      #
-      url: "api.crowdstrike.com"
+# If the installation method is 'url', provide the url for the sensor to
+# be downloaded.
+#
+falcon_download_url:
 
-      #
-      # 'Client ID' and 'Client Secret' for authentication against the CrowdStrike
-      # API. If unknown, download from the CrowdStrike support portal:
-      #
-      #   https://falcon.crowdstrike.com/support/api-clients-and-keys
-      #
-      client_id: ""
-      client_secret: ""
+# If using 'api' or 'url' methods, the number of retries the
+# installer should attempt.
+#
+falcon_retries: 3
 
-falconctl:
-    aid: ""
-    apd: ""
-    aph: ""
-    app: ""
-    trace: ""
-    feature: ""
-    message_log: ""
-    billing: ""
-    assert: ""
-    memfail_grace_period: ""
-    memfail_every_n: ""
+# If using 'api' or 'url' methods, the time (in seconds) the installer
+# should wait between retries.
+falcon_delay: 3
```

Comment:
FWIW these numbers were totally made up in the original code. Was more a reflection of impatience, not wanting to wait 10 seconds, before an error. If you think they should be changed then please do update these values! :)

Reply:
@shawndwells they actually seem reasonable to me.